web/packages/design/src/Button/Button.tsx

```diff
@@ -409,5 +407,12 @@ export const ButtonWarningBorder = <E extends React.ElementType = 'button'>(
   props: ButtonProps<E>
 ) => <Button fill="border" intent="danger" {...props} />;
 export const ButtonText = <E extends React.ElementType = 'button'>(
-  props: ButtonProps<E>
-) => <Button fill="minimal" intent="neutral" {...props} />;
+  props: ButtonProps<E> & { compact?: boolean }
+) => (
+  <Button
+    fill="minimal"
+    intent="neutral"
+    {...(props.compact ? { px: 1 } : {})}
+    {...props}
+  />
+);
```

Comment:
@bl-nero i took your compact suggestion, is this what you were envisioing?

Reply:
@kimlisa I was thinking about "same padding as with `inputAlignment` (which would be `px: 2`), unless `size=small`", but if an even smaller padding is commonly required, I have no objection here.

I would, however, declare the prop on the `Button` itself. My reason is that these additional components should be just wrappers, i.e. I should be equally well able to use `<Button fill="minimal" intent="neutral" compact/>` and achieve the same result.

Reply:
how about this: https://github.com/gravitational/teleport/pull/45241/commits/02f61df5dff1402a2eef2d8e7c36aee606911752#diff-a5c3729e6a710232f16bb4cbd373a966923e36fd2c6ade283ff0babef859f304R64

Reply:
LGTM